src/controllers/router/index.tsx

```diff
@@ -15,13 +15,13 @@ export const Router = ({
   onPrefetch,
   routes,
 }: RouterProps) => {
-  useEffect(() => {
-    const { unlisten } = getRouterState();
+  const { unlisten } = getRouterState();
```

Comment:
Fine, even if I think the problem is the use of `getRouterState` which is bad (as out of react control flow)